.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "daily"
+    time: "11:00"
+  open-pull-requests-limit: 10
+  assignees:
+    - "dependabot"
+
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
@jsjoeio Are you OK sticking with daily? I have a preference for daily, but if you feel strongly about it, then I'm happy to go with weekly

Reply:
Well they're being assigned and reviewed by you so if you think you can handle daily, all you 😛

Reply:
Putting this here so we can have a threaded conversation. cc @code-asher 

So, I think this is sorta blocked by #2559. I've learned recently that [Dependabot PR runs in Actions won't receive secrets](https://docs.github.com/en/github/administering-a-repository/keeping-your-actions-up-to-date-with-dependabot#about-dependabot-version-updates-for-actions):

> **Note**: Pull requests for Dependabot version updates will trigger workflow runs with a read-only `GITHUB_TOKEN`. These workflow runs will not be granted access to any secrets.

However, we found in `cdr/coder-cli` that a workaround is to manually re-run the Dependabot PR builds, in which case the run will get secrets since it seems to use the credentials of the person triggering the re-run. This seems like sensible - though non-obvious - behavior.

So I don't know if this is still worth pursuing given the required workaround, or not. What are your thoughts?

Reply:
Personally if it was me reviewing the dependabot PRs I wouldn't mind re-running manually for now until we make the required CI fixes. If anything I'd consider it motivation to get CI fixed. :wink: